styles/pages/Home.module.scss

```diff
@@ -124,15 +128,25 @@
     }
   }
 
+  .aboutUsContainer {
+    display: flex;
+    flex-direction: row;
+  }
+
   .aboutUs {
     display: flex;
+    flex-direction: row;
```

Comment:
same as above

Reply:
Resolved in commit 607f5a99ec73425562b5fd8c58e46dd6586477d1